src/app/course-home-page/all-links/all-links.component.css

```diff
@@ -0,0 +1,41 @@
+.gary-box{
+    background-color: #413c3c23;
+    padding: 20px;
+    overflow-x: hidden;
+    
+    
+}
+.toggleItem{
+    display: block;
+}
+@media(min-width:768px){
```

Comment:
we are using sass and not css. So you will have to change the way media queries are wirtten. Ask Manikya about this to point you to sass media queries link